deblur/deblurring.py

```diff
@@ -55,7 +65,31 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
+def create_error_profile(read_error):
+    """Create a theoretical read error profile given
+    a maxima error rate
+
+    Parameters
+    ----------
+    read_error : float
+        the upper bound on the hamming one read error
+
+    Results
+    -------
+    error_profile : list of float
+        the predicted theoretical error profile
+        (note for hamming>1 we take the max of
+        read_error^2 and 0.01 since closer to reality)
+    """
+    error_profile = [1.0, read_error, np.max([0.01, np.power(read_error, 2)]),
```

Comment:
I think it is ok to ditch `create_error_profile()` in this PR, since that is going to be kicked out anyways.